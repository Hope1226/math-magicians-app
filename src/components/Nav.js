import React from 'react';
import { Link } from 'react-router-dom';
import './stylesheets/nav.css';

const Nav = () => {
  const linkStyle = {
    color: 'white',
    textDecoration: 'none',
    fontWeight: 600,
    fontSize: '20px',
  };

  const heroText = {
    fontSize: '40px',
    fontWeight: 900,
  };

  return (
    <nav>
      <h1 style={heroText}>Math Magicians</h1>
      <ul>
        <Link style={linkStyle} to="/">
          <li>Home</li>
        </Link>
        <Link style={linkStyle} to="/calculator">
          <li>Calculator</li>
        </Link>
        <Link style={linkStyle} to="/quote">
          <li>Quote</li>
        </Link>
      </ul>
    </nav>
  );
};

export default Nav;
