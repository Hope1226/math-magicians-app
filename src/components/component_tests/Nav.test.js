import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';
import Nav from '../Nav';

describe('Loads the Nav page', () => {
  it('and matches the snapshot', () => {
    const tree = renderer.create(
    <MemoryRouter>
      <Nav />
    </MemoryRouter>
    ).toJSON;
    expect(tree).toMatchSnapshot();
  })
})