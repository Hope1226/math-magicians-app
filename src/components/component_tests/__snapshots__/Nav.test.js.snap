// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loads the Nav page and matches the snapshot 1`] = `[Function]`;
